src/editors/array.js

```diff
@@ -456,7 +456,9 @@ JSONEditor.defaults.editors.array = JSONEditor.AbstractEditor.extend({
     
     // Buttons to delete row, move row up, and move row down
     if(!self.hide_delete_buttons) {
-      self.rows[i].delete_button = this.getButton(self.getItemTitle(),'delete','Delete '+self.getItemTitle());
+      self.rows[i].delete_button = this.getButton(
+        self.getItemTitle(),'delete',
+        this.translate('Delete ')+self.getItemTitle());
```

Comment:
This is not good, please use the `variables` argument instead to allow for reordering the item title position.